transaction.go

```diff
@@ -5,30 +5,48 @@ package gosnowflake
 import (
 	"context"
 	"database/sql/driver"
+	"errors"
 )
 
 type snowflakeTx struct {
 	sc  *snowflakeConn
 	ctx context.Context
 }
 
-func (tx *snowflakeTx) Commit() (err error) {
-	if tx.sc == nil || tx.sc.rest == nil {
-		return driver.ErrBadConn
-	}
-	_, err = tx.sc.exec(tx.ctx, "COMMIT", false /* noResult */, false /* isInternal */, false /* describeOnly */, nil)
-	if err != nil {
-		return
+type txCommand int
+
+const (
+	commit txCommand = iota
+	rollback
+)
+
+func (cmd txCommand) string() (txStr string, err error) {
+	switch cmd {
+	case commit:
+		return "COMMIT", nil
+	case rollback:
+		return "ROLLBACK", nil
 	}
-	tx.sc = nil
-	return
+	return "", errors.New("unsupported transaction command")
+}
+
+func (tx *snowflakeTx) Commit() (err error) {
```

Comment:
Do we need the return value name? I think we can drop `err` and parenthesis?

Reply:
True, applied suggestion.